src/main/java/ua/danit/final_project/entities/WorkShift.java

```diff
@@ -0,0 +1,32 @@
+package ua.danit.final_project.entities;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "work_shift")
+@Data
+public class WorkShift {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @ManyToOne
+  @JoinColumn(name = "u_id", nullable = false)
+  private User user;
+
+  @ManyToOne
+  @JoinColumn(name = "s_id", nullable = false)
+  private Schedule schedule;
+
+  private Timestamp date;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "date" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=59&issues=AWUy2H-mW2AOP0Ck_lbT&open=AWUy2H-mW2AOP0Ck_lbT)

[ISSUE_KEY]: AWUy2H-mW2AOP0Ck_lbT